src/Object/Contact.php

```diff
@@ -746,8 +720,7 @@ public static function getPostsFromUrl($contact_url)
 
 		// There are no posts with "uid = 0" with connector networks
 		// This speeds up the query a lot
-		$r = q("SELECT `network`, `id` AS `author-id`, `contact-type` FROM `contact`
-		WHERE `contact`.`nurl` = '%s' AND `contact`.`uid` = 0", dbesc(normalise_link($contact_url)));
+		$r = dba::select('contact', array('network', 'id AS author-id', 'contact-type'), array('nurl' => normalise_link($contact_url), 'uid' => 0));
```

Comment:
Please use `dba::inArray()` to get the same behavior as `q()`.
Either use `limit => 1` or `dba::fetch($r)`, `$r[0]` is invalid with `dba::select()`